Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -3,38 +3,64 @@
 .bocCheckBox
 {
   margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox img
+{
+  margin: 2px;
   vertical-align: middle;
+  height: 16px;
+  width: 16px;
```

Comment:
See the next statements on input. question, should we apply both selectors to the same declaration?